### DISCUSSION TITLE
Donations & Contributions to NaturalCrit

### DISCUSSION THREAD

**Discussion by Marcobertini6** [naturalcrit/homebrewery#3794] [Ideas]
hello, 
in recent months I have used a lot of homebrewery and wanted to find a way to thank you. Is there a way to donate something to the creator or staff?

**Comment by xbnrxout** [comment]
They have a patreon
https://www.patreon.com/Naturalcrit

**Comment by Marcobertini6** [comment]
thank you, is there a way to make a single donation?

**Comment by EmmanueleVilla** [comment]
> thank you, is there a way to make a single donation?

just subscribe and then unsubsribe

**Comment by calculuschild** [comment]
@EmmanueleVilla Thanks for your willingness to support us!

**Comment by ericscheid** [comment]
There has been [additional interest](https://www.reddit.com/r/homebrewery/comments/sq2o3h/is_there_a_non_patreon_way_to_support/) in one-off donations (which doesn't involve an unintuitive sub/unsub process)

We could set up a ko-fi account, or set up [GitHub Sponsor](https://github.com/sponsors/ericscheid/waitlist) (there might be a wait-list).

- [ ] decide which way forward: ko-fi, GH sponsors, both?
- [ ] set up chosen thing(s)
- [ ] add link to the "Helping out" section on home page

**Comment by dbolack-ab** [comment]
Bug bounties?

**Comment by calculuschild** [comment]
> Bug bounties?

Do you have a recommended bug bounty system?

**Comment by dbolack-ab** [comment]
> > Bug bounties?
> 
> Do you have a recommended bug bounty system?

I don't have any opinions on any of them - seemed like a simple alternate method for folks to one-off.
